### ISSUE TITLE
Use `Repl` with `usePGLite` with typescript

### ISSUE THREAD

**Issue by AntonOfTheWoods** [electric-sql/pglite#207]
I get the ts error:

```
Type 'PGliteWithLive' is missing the following properties from type 'PGlite': #private, Modulets(2739)
```

Most things seem to work if I simply do a:

```
        <Repl
          // @ts-ignore
          pg={db}
        />
```

Is `Repl` *supposed* to work with `usePGLite`?

**Comment by samwillis** [comment]
Yep, we want it too, thats a bug. We haven't actually tested the new React hooks with the Repl component.

Looks like it's just a type issue, as you've noted `@ts-ignore` makes it work.

**Comment by samwillis** [comment]
This is fixed the just published v0.2.1
